src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/RazorDocumentManager.ts

```diff
@@ -198,6 +198,8 @@ export class RazorDocumentManager implements IRazorDocumentManager {
             csharpProjectedDocument.update(updateBufferRequest.changes, updateBufferRequest.hostDocumentVersion);
 
             this.notifyDocumentChange(document, RazorDocumentChangeKind.csharpChanged);
+        } else {
+            this.logger.logMessage('Failed to update the C# document buffer. This is unexpected and may result in incorrect C# interactions.')
```

Comment:
Should this be `logWarning` or `logError`?

Reply:
Ah, definitely warning 👍